### PR TITLE
fix a bug that using `Instance::destroy()` under multiple primary keys

### PR DESCRIPTION
**Pull request by XadillaX**
If the instance is under multiple primary keys, the `where` section of query string should using multiple primary keys.

### CODE REVIEW

**Comment by janmeier** [comment]
Needs some integration tests, otherwise looks good

**Comment by mickhansen** [comment]
This could probably benefit from: https://github.com/sequelize/sequelize/pull/3701/files#diff-a5b70841c2143e5d2e0fd98dd3b4cefeR75

**Comment by XadillaX** [comment]
@janmeier But I see there's no previous integration tests for `destroy()`, only tests for `update()`.

**Comment by mickhansen** [comment]
https://github.com/sequelize/sequelize/blob/master/test/integration/instance.test.js#L1697

**Comment by XadillaX** [comment]
added. @mickhansen @janmeier

File: test/integration/instance.test.js [hunk 1]
```diff
@@ -1759,6 +1759,38 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
...
+          primaryKey: true
+        }
+      });
+
+      return MultiPrimary.sync({ force: true }).then(function() {
+        return MultiPrimary.create({ bilibili: 'bl', guruguru: 'gu' }).then(function() {
+          return MultiPrimary.create({ bilibili: 'bl', guruguru: 'ru' }).then(function(m2) {
+            return MultiPrimary.findAll().then(function(ms) {
+              expect(ms.length).to.equal(2);
+              return m2.destroy({
```
Comment (mickhansen) [inline test/integration/instance.test.js]: We prefer not to test the SQL in integration tests (older tests that do that are slowly being ripped out).
Can you please refactor the test to check the expected result of a findAll after the destroy?

**Comment by XadillaX** [comment]
@mickhansen Okay, finished.

**Comment by mickhansen** [comment]
Thank you @XadillaX

**Comment by XadillaX** [comment]
You're welcome. Thank you for the awesome project.

**Pull request merged**